lib/StreamingStore.ts

```diff
@@ -5,6 +5,15 @@ import type { Readable } from 'readable-stream';
 import { PassThrough } from 'readable-stream';
 import { PendingStreamsIndex } from './PendingStreamsIndex';
 
+interface ILocalStore<Q extends RDF.BaseQuad> extends RDF.Store<Q> {
+  countQuads: (
+    subject: RDF.Term | null,
+    predicate: RDF.Term | null,
+    object: RDF.Term | null,
+    graph: RDF.Term | null,
```

Comment:
```suggestion
    subject?: RDF.Term | null,
    predicate?: RDF.Term | null,
    object?: RDF.Term | null,
    graph?: RDF.Term | null,
```

Reply:
n3 store doesn't support `undefined`: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4c017214b1c3fcfd5bb1637035f806b06c6b76d3/types/n3/index.d.ts#L295 the [OTerm](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4c017214b1c3fcfd5bb1637035f806b06c6b76d3/types/n3/index.d.ts#L128C13-L128C18) can only be a `RDF.Term`, string or null.

Reply:
It actually does support `undefined`: https://github.com/rdfjs/N3.js/blob/main/src/N3Store.js#L445-L447
But it appears the typings are not fully correct.

So yeah, let's keep it as-is for now. We can modify later once the N3 typings would get fixed.

Reply:
I assume this is support for when [OTerm](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/4c017214b1c3fcfd5bb1637035f806b06c6b76d3/types/n3/index.d.ts#L128C13-L128C18) is `null` but the logic is the same for `undefined`.

Reply:
Indeed, but since `null` is not the same as `undefined`, tsc will complain about it.